modules/ROOT/pages/8.2.0-release-notes.adoc

```diff
@@ -29,6 +29,29 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 The following premium plugin updates were released alongside {productname} 8.2.0.
 
+=== Media Optimizer
+
+The {productname} 8.2.0 release includes an accompanying release of the **Media Optimizer** premium plugin.
+
+**Media Optimizer** now includes comprehensive video handling capabilities.
+
+==== Video Upload Support
+
+The **Media Optimizer** plugin now supports video upload and processing, extending beyond its previous image-only functionality. Users can now upload videos directly within the editor through drag-and-drop, file selection, or URL input.
+
+The plugin now supports multiple video formats:
+
+* **MP4 Video**: Full support for MP4 video files `video/mp4`
+* **WebM Video**: Support for WebM video files `video/webm`
```

Comment:
this should be a more generic thing since it will support a lot of formats, check TINY-12881 under `documentation`

Reply:
We need to be explicit in what is supported and what is not, leaving this "blank" does not help integrators to understand what is and what is not supported on upload.